src/types.ts

```diff
@@ -94,7 +94,7 @@ export interface BaseButton {
   /** Tells screen reader the controlled element is expanded */
   ariaExpanded?: boolean;
   /**
-   * @deprecated As of release 4.7.0, replaced by {@link https://polaris.shopify.com/components/structure/page#props-pressed}
+   * @deprecated As of release 4.7.0, replaced by {@link https://polaris.shopify.com/components/actions/button#prop-pressed}
```

Comment:
This was pointing to the wrong link. But also - as mentioned in my `Questions` section - should we not have removed this already?

Reply:
Good spot! Yeah this should have ideally been removed in v5.0.0 but we missed that window. We should try again in v6.0.0